e2e-tests/build

```diff
@@ -9,30 +9,31 @@ src_dir="$(realpath ${test_dir}/..)"
 IMAGE=$(echo "$IMAGE" | sed -e 's#percona/#perconalab/#')
 
 build_operator() {
-    if [ "${RELEASE:-1}" = 0 ]; then
-        GO_LDFLAGS="-race"
-    fi
+	if [ "${RELEASE:-1}" = 0 ]; then
+		GO_LDFLAGS="-race"
+	fi
 
-    export IMAGE
-    export GO_LDFLAGS="-w -s -trimpath $GO_LDFLAGS"
-    pushd ${src_dir}
-        docker build \
-            --build-arg GIT_COMMIT=$GIT_COMMIT \
-            --build-arg GIT_BRANCH=$GIT_BRANCH \
-            --build-arg GO_LDFLAGS="$GO_LDFLAGS" \
-            --squash \
-            --no-cache \
-            -t "${IMAGE}" -f build/Dockerfile .
-    popd
-    docker push ${IMAGE}
+	export IMAGE
+	export GO_LDFLAGS="-w -s -trimpath $GO_LDFLAGS"
+	pushd ${src_dir}
+	docker build \
+		--build-arg GIT_COMMIT=$GIT_COMMIT \
+		--build-arg GIT_BRANCH=$GIT_BRANCH \
+		--build-arg BUILD_TIME=$BUILD_TIME \
```

Comment:
is `$BUILD_TIME` defined somewhere?
I mean, will it be not empty if do `./e2e-tests/build`?

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/803/commits/005769b7b3bec4e99122bfbcc1258d6c260723fb
Checked it out accidentally